meta/subtype-schemas.json

```diff
@@ -232,6 +232,12 @@
                 }
             }
         },
+        "ml-model": {
+            "type": "object",
+            "subtype": "ml-model",
+            "title": "Machine Learning Model",
+            "description": "A machine learning model, accompanied with STAC metadata that implements the the STAC ml-model extension."
```

Comment:
> accompanied with STAC metadata that implements the the STAC ml-model extension

What does this practically mean here in this context of defining a JSON schema? Isn't that more a concern of a process like `save_ml_model` that actually "exports" the model to a more concrete form?

Reply:
This was just to remove the error due to ml model being returned.

Reply:
> 



> > accompanied with STAC metadata that implements the the STAC ml-model extension
> 
> What does this practically mean here in this context of defining a JSON schema? Isn't that more a concern of a process like `save_ml_model` that actually "exports" the model to a more concrete form?

This was to remove the error due to the ml model being returned. I created a branch from the draft and not the ml branch.